code/modules/research/mechfab_designs.dm

```diff
@@ -1038,4 +1038,151 @@
 	time = 20
 	req_tech = list(TECH_MATERIAL = 6, TECH_ENGINEERING = 5, TECH_PHORON = 3, TECH_MAGNET = 4, TECH_POWER = 6)
 	materials = list(DEFAULT_WALL_MATERIAL = 10000, "glass" = 6000, "silver" = 4000)
-	
\ No newline at end of file
+
+// Exosuit Internals
+
+/datum/design/item/mechfab/exointernal
+	category = "Exosuit Internals"
+	time = 120
+	req_tech = list(TECH_MATERIAL = 3, TECH_ENGINEERING = 3)
+
+/datum/design/item/mechfab/exointernal/stan_armor
+	name = "Armor Plate (Standard)"
+	category = "Exosuit Internals"
+	id = "exo_int_armor_standard"
+	req_tech = list(TECH_MATERIAL = 2, TECH_ENGINEERING = 2)
+	materials = list(DEFAULT_WALL_MATERIAL = 10000)
+	build_path = /obj/item/mecha_parts/component/armor
+
+/datum/design/item/mechfab/exointernal/light_armor
+	name = "Armor Plate (Lightweight)"
+	category = "Exosuit Internals"
+	id = "exo_int_armor_lightweight"
+	req_tech = list(TECH_MATERIAL = 2, TECH_ENGINEERING = 3)
+	materials = list(DEFAULT_WALL_MATERIAL = 5000, MAT_PLASTIC = 3000)
+	build_path = /obj/item/mecha_parts/component/armor/lightweight
+
+/datum/design/item/mechfab/exointernal/reinf_armor
+	name = "Armor Plate (Reinforced)"
+	category = "Exosuit Internals"
+	id = "exo_int_armor_reinforced"
+	req_tech = list(TECH_MATERIAL = 4, TECH_ENGINEERING = 4)
+	materials = list(DEFAULT_WALL_MATERIAL = 20000, MAT_PLASTEEL = 10000)
+	build_path = /obj/item/mecha_parts/component/armor/reinforced
+
+/datum/design/item/mechfab/exointernal/mining_armor
+	name = "Armor Plate (Blast)"
+	category = "Exosuit Internals"
+	id = "exo_int_armor_blast"
+	req_tech = list(TECH_MATERIAL = 4, TECH_ENGINEERING = 4)
+	materials = list(DEFAULT_WALL_MATERIAL = 20000, MAT_PLASTEEL = 10000)
+	build_path = /obj/item/mecha_parts/component/armor/mining
+
+/datum/design/item/mechfab/exointernal/gygax_armor
+	name = "Armor Plate (Marshal)"
```

Comment:
Is there anything explaining what these different armor types do and which mechs they're actually designed for/based on?

Reply:
Marshal is in reference to the Police / Corp Sec exosuit, Gygax. Military is the Durand.